### PR TITLE
Show SurveyToast every 60 active days

### PR DESCRIPTION
**Pull request by christinaforney**
We would like to get more frequent feedback from our users. Currently, we only show the survey toast on the third day of using the product. This means that we're asking for feedback early in their usage, but aren't following up to see if that changes over time. We don't want to bother users too frequently, so only showing each 60 days of usage will make sure it is not happening too frequently and will likely be a frequency of ~1x/quarter. 

- Show survey toast on all pages (not just a code view page) to avoid users missing it by not going to the right page that day
- Show survey toast every 60 days of activity

There are longer term improvements that I would like to make here, but this is a quick win to allow users to share feedback with us more regularly.

### CODE REVIEW

File: web/src/Layout.tsx [hunk 1]
```diff
@@ -130,6 +131,7 @@ export const Layout: React.FunctionComponent<LayoutProps> = props => {
             {!needsSiteInit && !isSiteInit && !!props.authenticatedUser && (
                 <IntegrationsToast history={props.history} />
             )}
+            {!isSiteInit && <SurveyToast authenticatedUser={props.authenticatedUser} />}
```
Comment (dadlerj) [inline web/src/Layout.tsx]: Any chance https://github.com/sourcegraph/sourcegraph/issues/6028 hits us here too? Shouldn't block merging this, but worth following up on cc @attfarhan

Comment (christinaforney) [reply web/src/Layout.tsx]: I _think_ that is what `!isSiteInit` does, but would love @attfarhan to confirm!

Comment (dadlerj) [reply web/src/Layout.tsx]: Haha 🤦‍♂ yeah that seems like it

Comment (attfarhan) [reply web/src/Layout.tsx]: Yep, that's correct 👍 !isSiteInit handles it

**Pull request merged**